v4.txt

```diff
@@ -110,16 +130,41 @@ Example response: {"num_tokens":78,"tokenizer":"cl100k_base","usage":{"tokens":0
 Note: for the API to return chunks, you must specify `"return_chunks": true` as part of the request body.
 
 7. Classifier API
+a) Classify
```

Comment:
I think we should just use zero shot classifier in the prompt 🤔 because it has better performance overall, we only need to suggest using semantically meaningful labels. This way train/delete/list APIs will also not be necesary, wdyt?

Reply:
okay. Makes sense. Otherwise we have too many failure cases